src/statements.md

```diff
@@ -36,7 +36,12 @@ declaration until the end of the enclosing block scope.
 
 ## Expression statements
 
-An _expression statement_ is one that evaluates an [expression](expressions.html)
-and ignores its result. The type of an expression statement `e;` is always
-`()`, regardless of the type of `e`. As a rule, an expression statement's
-purpose is to trigger the effects of evaluating its expression.
+An _expression statement_ is one that evaluates an
+[expression](expressions.html) and ignores its result. The type of an
+expression statement `e;` is always `()`, regardless of the type of `e`. As a
+rule, an expression statement's purpose is to trigger the effects of evaluating
+its expression. [Block expressions](expressions.html#block-expressions) and
+control flow expressions, such as [`loop`
+expressions](expressions.html#infinite-loops) and [`if`
+expressions](expressions.html#if-expressions), which evaluate to `()` can be
+used as statements without a trailing semicolon.
```

Comment:
Should this be:

> ...can be used as expressions statements...

Or are we using 'expression statement' and 'statement' interchangeably here?